### DISCUSSION TITLE
Is there a way to force the Worker to use reverse when working in the corner's headlands?

### DISCUSSION THREAD

**Discussion by guerrillian** [Courseplay/Courseplay_FS22#2617] [Q&A]
So I'd like to force the worker to use reverse in order to avoid his mistakes in the corners.

I already use the sharp mode, but I'd like even more if it exists.

Thanks!

**Comment by guerrillian** [comment]
I'd like to force it:

As you can see, the roller does it, the planer doesn't.

![image](https://github.com/Courseplay/Courseplay_FS22/assets/128705907/e92979d1-b95f-4bb2-85d7-b5cd8c4fc487)

**Comment by Tensuko** [comment]
So is there a sharp turn?
I would need to see the course waypoints.
If that is the case, what you can do is use the setting lower implement and set it to early.

**Comment by guerrillian** [comment]
you solved it again @Tensuko.

Thanks, again.

Here what I did:
![image](https://github.com/Courseplay/Courseplay_FS22/assets/128705907/06f0b865-eed4-4674-b460-6a0a52910fab)
